drivers/spi/spi_mcux_lpspi.c

```diff
@@ -386,6 +438,40 @@ static int wait_dma_rx_tx_done(const struct device *dev)
 	}
 }
 
+static int spi_mcux_dma_rxtx_load(const struct device *dev,
+				size_t *dma_size)
```

Comment:
why is this a pointer

Reply:
The size variable is used for functions that directly follow this function in our current implementation of the driver.

Reply:
yes I missed where you wrote to it, makes sense